projects/core/src/lib/util/components/not-found/not-found.component.html

```diff
@@ -0,0 +1 @@
+<h2>This route is not on the map :(</h2>
```

Comment:
Please add our look-and-feel here

Reply:
part of [99a0802](https://github.com/geo-engine/geoengine-ui/pull/512/commits/99a080258fa877d66157cf1184739c316b901dc2)